cfme/physical/provider/__init__.py

```diff
@@ -133,6 +133,8 @@ class Details(CFMENavigateStep):
 
     def step(self):
         self.prerequisite_view.entities.get_entity(name=self.obj.name, surf_pages=True).click()
+        if self.view.toolbar.view_selector.is_displayed:
```

Comment:
Optional: I'd rather move this statement to resetter until you start using Dashboard.
```
    def resetter(self):
        """Reset view and selection"""
        view_selector = self.view.toolbar.view_selector
        if view_selector.selected != 'Summary View':
            view_selector.select('Summary View')
```